smartcontractkit/external-adapters-js
# chore(deps): update ipfs default imports to use ESM per ipfs-core-types@0.8.0

## Closes #25095

## Description
IPFS release [ipfs-core-types@0.8.0](https://github.com/ipfs/js-ipfs/releases/tag/ipfs-core-types%400.8.0) requires clients to use specific exports rather than default exports. This PR updates the imports in packages/sources/ipfs/src/endpoint/read.ts from `IPFS` to `IPFSHTTPClient`.

......

## Changes

- Update `IPFS` imports to `IPFSHTTPClient` imports in packages/sources/ipfs/src/endpoint/read.ts

## Steps to Test

1. Install IPFS CLI (For Mac Homebrew users: `brew install ipfs`, for others: https://docs.ipfs.io/install/command-line/)
2. Start IPFS node (see https://docs.ipfs.io/how-to/command-line-quick-start/#prerequisites)
2. Import a file through the [IPFS node web UI](localhost:5001/webui) (Go to Files > +Import)
3. Copy the imported file's CID
![Screen Shot 2022-01-18 at 11 06 33 AM](https://user-images.githubusercontent.com/3673544/150002420-a26b1caf-a955-43f9-8669-85fb374aea79.png)
4. Start the IPFS external adapter (It should point to the local IPFS node by default)
5. cURL the external adapter with `type:raw` using the CID copied above (e.g. `curl -H "Content-Type: application/json" -X POST -d '{"id":"2","data":{"cid":"QmPqgFjkyTSaxxmZ2DPZgZStz2tHC82aFnYcNs77vfjFM2","type":"raw"}}' http://localhost:8080`)
6. Verify the request is successful with status:200
7. cURL the external adapter with `type:dag` using the CID copied above (e.g. `curl -H "Content-Type: application/json" -X POST -d '{"id":"2","data":{"cid":"QmPqgFjkyTSaxxmZ2DPZgZStz2tHC82aFnYcNs77vfjFM2","type":"dag"}}' http://localhost:8080`)
8. Verify the request is successful with status:200
9. Compare outputs from previous steps with outputs from `master` or `develop` branches

## Quality Assurance
